driver/driver_config.h.in

```diff
@@ -13,3 +13,5 @@ or GPL2.txt for full copies of the license.
 #define PROBE_NAME "${PROBE_NAME}"
 
 #define PROBE_DEVICE_NAME "${PROBE_DEVICE_NAME}"
+
+#define KBUILD_MODNAME PROBE_DEVICE_NAME
```

Comment:
Not sure this is enough, this is getting passed to a macro composition down in the kernel, we probably need to replace dashes and other special symbols before letting this go down. But feel free to ignore me if I’m wrong, I didn’t put my head on this yet